src/System.ServiceModel.Http/src/System.ServiceModel.Http.builds

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.ServiceModel.Http.csproj" />
+    <Project Include="System.ServiceModel.Http.csproj" >
+      <TargetGroup>net46</TargetGroup>
+    </Project>
+    <Project Include="System.ServiceModel.Http.csproj" >
```

Comment:
Do you want this project to build for every OSGroup or just Windows_NT? If you want just Windows_NT then keep this entry and delete the first Project in this item group if you want every then delete this one and keep the first one.